svc/IdleService.ts

```diff
@@ -15,20 +16,30 @@ import {MINUTES, olderThan} from '@xh/hoist/utils/datetime';
  * system from unattended clients and/or as a "belt-and-suspenders" defence against memory
  * leaks or other performance issues that can arise with long-running sessions.
  *
- * This service consults the `xhIdleConfig` soft-config and the `xh.disableIdleDetection`
+ * This service consults the `xhIdleConfig` soft-config and the `xh.xhIdleDetectionDisabled`
  * user preference to determine if and when it should suspend the app.
+ *
+ * This service also exposes the observable property `idleFor` which holds the number of minutes
+ * that the app has been idle.  Apps that do not want to use Hoist's app suspension (maybe because
+ * it will cause users to lose unsaved state) can set up their own "lighter touch" shutdown of resource
+ * intensive recurring data calls at whatever threshold of `idleFor` needed.  Apps can resume those
+ * calls when the `idleFor` value becomes 0 as activity is detected.
  */
 export class IdleService extends HoistService {
     override xhImpl = true;
 
     static instance: IdleService;
 
+    @observable idleFor: number = 0;
```

Comment:
Need to add some sense of the units in here - looks like this is in minutes? 

If we keep it at minutes, we should consider if we want it reporting 1min when idle for 30 secs, and how often we need to run the timer, how to make it clear what the contract is.